apps/heureka/src/components/Services/utils.ts

```diff
@@ -134,6 +138,14 @@ export const getActiveServiceFilter = (filterSettings: FilterSettings): ServiceF
     }, {}),
 })
 
+type IssueCounts = {
```

Comment:
same here. We are duplicating the types...
https://github.com/cloudoperators/juno/blob/main/apps/heureka/src/components/Services/useFetchServicesCounts.ts#L15